ristonrodrigues723.md

```diff
@@ -0,0 +1,172 @@
+ 
+will make use of:
+- 1 x ESP32 board- main part of the module to connect to mifi and carry out processing
+- 1 x | GL-12 840 Points Solderless Breadboard  | - the breadbords will be used here and one other module,
+- 1 x | 20 CM 40 Pin Dupont Cable Male/Male, Male/Female, Female/Female Cable Combo | will connect the breadboards to other parts such as sensors and electronic components and will be uised in different modules.
+- 1 x | MQ-2 Smoke LPG Butane Hydrogen Gas Sensor Detector Module |- gas sensor to sense 
+- 1 x | Tactile Push Button Switch set | buttons will be  connected in the module to stop the module  in case of server failure.
+- 1 x | D10 3.7V Rechargeable Lithium ion Battery Power Bank Cell | -An external battery to connect to the ESP32 
+- 1 x |Flame Sensor Module |- connected to the esp 32 board.
+
+
+8) garden module -used to pay attention  to soil condition and weather in a garden
+
+will make use of:
+- 1 x| Electronics ESP32 ESP-32 ESP-32S ESP 32 Development Board CP2102 WiFi Bluetooth Ultra-Low Power Consumption Dual Core  |
+- 1 x  Waveshare Moisture Sensor -to measure moisture in the environment will connect to esp 32
+- 1 x | GL-12 840 Points Solderless Breadboard  |- for mounting the componemnts - wires ,sensors
+- 1 x | Female to Female DuPont Line 40 Pin 30cm | - used to connect the circut
+- 1 x | GL-12 840 Points Solderless Breadboard |
+- 1 x | Raindrops Detection Sensor Module	 |- used to me
+- 2 x |Soil Moisture Meter, Soil Humidity Sensor, Water Sensor, Soil Hygrometer for Ardunio  | will be positioned in the garden in different will be connected to esp 32 boards
+ 
+ 
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| ApTechDeals 60W /100 Watt Dual Watt Glue Gun | [Supplier/Link                   ](https://www.amazon.in/gp/product/B076CRTD7H/ref=ox_sc_act_image_2?smid=A2K5VGK3OPN02U&th=1)      | $7.86   |
+| GLUN 11 mm Transparent HOT MELT glue sticks       |    https://www.amazon.in/gp/cart/view.html?ref_=nav_cart  | $2.24   |
+| Generic E-DIMMER AC 50-220V 2000W SCR Triac Electric Voltage Regulator Temperature/Motor Speed Controller Light Dimmer X4| https://www.amazon.in/50-220V-Electric-Regulator-Temperature-Controller/dp/B07GXG82HC/ref=sr_1_9?ascsubtag=dc6cf6efbd9e870c0a7c0b6b20716b95&keywords=light+dimmer&qid=1672399537&sr=8-9   | $8.22   |
+| D10 3.7V Rechargeable Lithium ion Battery Power Bank Cell  X 3/[Product         ](https://www.amazon.in/Rechargeable-Lithium-Bluetooth-Emergency-3-7v/dp/B0BKLJW6WZ/ref=sr_1_3?ascsubtag=2b0d051f60e31b9869fd994ea8086665&keywords=lithium-ion+battery&qid=1672400083&sr=8-3)| Supplier/Link                         | $11.0  |
```

Comment:
how would you connect this cell to the esp32? do you need a connector? and, would you need a 5v battery instead? https://techexplorations.com/guides/esp32/begin/power/

Reply:
he connected it directly using wires and I have a 5v battery from an old module i can use that

Reply:
the dupoint wires and soldering will help

Reply:
ok cool! do you already have dupont wires that are solder compatible?

Reply:
ok cool!